src/PostgREST/Error.hs

```diff
@@ -285,6 +288,12 @@ instance JSON.ToJSON ApiRequestError where
   toJSON (ColumnNotFound relName colName) = toJsonPgrstError
     SchemaCacheErrorCode04 ("Could not find the '" <> colName <> "' column of '" <> relName <> "' in the schema cache") Nothing Nothing
 
+  toJSON (TableNotFound schemaName relName tbls) = toJsonPgrstError
+    SchemaCacheErrorCode05
+    ("Could not find the table or view '" <> schemaName <> "." <> relName <> "' in the schema cache")
```

Comment:
Looking at the error messages now, `table or view` only adds verbosity and it's not really helpful.

Additionally, I think API users should be kept in the dark about if the server uses a table or a view.

The most generic term would be "resource", but that will cause some head scratching.

So I think we should just go with "table" as the more generic term.

```suggestion
    ("Could not find the table '" <> schemaName <> "." <> relName <> "' in the schema cache")
```